src/server/src/main/java/io/cassandrareaper/core/EditableRepairSchedule.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021-2021 DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.cassandrareaper.core;
+
+import io.cassandrareaper.validators.NullOrNotBlank;
+import io.cassandrareaper.validators.ValidEditableRepairSchedule;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.apache.cassandra.repair.RepairParallelism;
+
+@Data
+@NoArgsConstructor
+@ValidEditableRepairSchedule
+public class EditableRepairSchedule {
+  @NullOrNotBlank
+  protected String owner;
+
+  @JsonProperty(value = "repair_parallelism")
+  protected RepairParallelism repairParallelism;
+
+  @Min(value = 0)
+  @Max(value = 1)
+  protected Double intensity;
+
+  @JsonProperty(value = "scheduled_days_between")
+  @Min(value = 0)
+  @Max(value = 31)
+  protected Integer daysBetween;
+
+  @JsonProperty(value = "segment_count_per_node")
+  @Min(value = 1)
```

Comment:
I was wrong in suggesting 1 as minimum value: if set to 0 it currently translates to the default number of segments which is applied when the repair run is created.
Currently it prevents from editing a schedule that had 0 segment per node if the value isn't set explicitly (which is then not consistent with the initial schedule creation).
You can remove the minimum bound to allow 0 as valid.
My bad for this.

Reply:
No worries at all, I updated the min to be 0 in the API and the UI.

Reply:
cool, thanks!